task/monorail-server/src/bsp/sidecar_bc.rs

```diff
@@ -225,12 +216,40 @@ impl<'a, R: Vsc7448Rw> Bsp<'a, R> {
         self.front_io_speed = [Speed::Speed1G; 2];
 
         self.phy_vsc8504_init()?;
-        self.phy_vsc8562_init()?;
 
         self.vsc7448.configure_ports_from_map(&PORT_MAP)?;
         self.vsc7448.configure_vlan_semistrict()?;
         self.vsc7448_postconfig()?;
 
+        // Some front IO boards have a faulty oscillator driving the PHY,
+        // causing its clock to misbehave some fraction of (re-)boots. Init
+        // the PHY in a loop, requesting the sequencer to reset as much as
+        // necessary to try and correct the problem.
+        let mut osc_good = false;
+
+        while !osc_good {
```

Comment:
This loop should only run if `self.vsc8562.is_some()`; otherwise, we'll get stuck here forever if the front IO board is missing.

Reply:
Ah yes, thank you!